frontend/packages/console-app/src/components/quick-starts/controller/QuickStartFooter.tsx

```diff
@@ -71,14 +102,36 @@ const QuickStartFooter: React.FC<QuickStartFooterProps> = ({
           onClick={onBack}
           isInline
         >
-          {t('quickstart~Back')}
+          {SecondaryButtonText.BACK}
+        </Button>
+      )
+    );
+
+  const getSideNoteAction =
+    status === QuickStartStatus.COMPLETE &&
+    (currentPath !== quickStartPath ? (
+      <Link style={{ display: 'inline-block' }} to={quickStartPath}>
+        {t('quickstart~View all tours')}
+      </Link>
```

Comment:
When does this link appear on Quick start?

Reply:
when /quickstart is not the URL like, quicksearch

Reply:
Do we know when / where this requirement came from (adding a `view all tours` link at the end of a quick start)
cc @bmignano @serenamarie125 

This should read `View all quick starts` if we keep it.

Reply:
I think we had that link in the very first iteration of quick starts (apparently back when they were still being called tours 😄) . It doesn't seem like it ever got in, but honestly I don't mind not having it. Thoughts @serenamarie125 ?

Reply:
Please remove the _View all tours_ link!